components/backup/src/endpoint.rs

```diff
@@ -860,16 +861,23 @@ fn get_max_start_key(start_key: Option<&Key>, region: &Region) -> Option<Key> {
     }
 }
 
-/// Construct an backup file name based on the given store id and region.
-/// A name consists with three parts: store id, region_id and a epoch version.
+/// Construct an backup file name based on the given store id, region, range start key and local unix timestamp.
+/// A name consists with five parts: store id, region_id, a epoch version, the hash of range start key and timestamp.
+/// range start key is used to keep the unique file name for file, to handle different tables exists on the same region.
```

Comment:
Can we remove the start key? Unix timestamp should be enough to generate a unique file name together with a store id.

Reply:
I'm not sure about that, because we do `backupRange` concurrently, if two backup jobs backup the same region with in same timestamp,  if this happened, it will cause the `already exists` error.🤔️